Makefile

```diff
@@ -7,8 +7,8 @@ WMCO_VERSION ?= 10.18.0
 
 # *_GIT_VERSION are the k8s versions. Any update to the build line could potentially require an update to the sed
 # command in generate_k8s_version_commit() in hack/update_submodules.sh
-KUBELET_GIT_VERSION=v1.31.1+1892e4d
-KUBE-PROXY_GIT_VERSION=v1.30.0
+KUBELET_GIT_VERSION=v1.31.1+8540bd4
+KUBE-PROXY_GIT_VERSION=v1.31.1
 CONTAINERD_GIT_VERSION=v1.7.20
```

Comment:
@saifshaikh48   should this bump containerd as well

https://github.com/containerd/containerd/releases/tag/v1.7.22

Reply:
hack script did not generate commit for containerd. from the submodule dir the version is v1.7.20 after running